notebooks/MAST/astroqueryMAST/beginner_search_astroqueryMAST/beginner_search_astroqueryMAST.ipynb

```diff
@@ -0,0 +1,395 @@
+   "metadata": {},
+   "source": [
+    "### 1. By Region\n",
+    "You can use `query_region` to find objects based on coordinates and radius.  \n",
+    "The coordinates can be given as a string or `astropy.coordinates` object, and the radius as a string or float object. If no radius is specified, the default is 0.2 degrees."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This will give a warning that the coordinates are being interpreted as an ICRS coordinate provided in degrees\n",
+    "obsByRegion = Observations.query_region(\"322.49324 12.16683\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# How many results? Let's look at the first ten.\n",
+    "print(\"Number of results:\",len(obsByRegion))\n",
+    "print(obsByRegion[:10])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2. By Object Name\n",
+    "\n",
+    "The `query.object` method allows you to search based on object name and optional radius argument.  \n",
+    "The object name is first resolved to sky coordinates, and then the resolved region is queried as above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "obsByName = Observations.query_object(\"M51\",radius=\".005 deg\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Number of results:\",len(obsByName))\n",
+    "print(obsByName[:10])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. By Other Criteria (with or without name/region) <a id=\"crit\"></a>\n",
+    "\n",
+    "To search for observations based on additonal parameters, you can use `query_criteria`. In a sense, this is a more powerful version of the tools above, as you can still search by coordinates and objectname; however, you can inculde additional desired criteria. You may also run the search without specifying a name or coordinates.\n",
+    "\n",
+    "To perform the search, give your critera as keyword arguments. Valid criteria are listed [here](https://mast.stsci.edu/api/v0/_c_a_o_mfields.html). Some examples are: \"coordinates\", \"t_exptime\" (exposure time), and \"s_dec\" (source declination)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "obsByCriteria = Observations.query_criteria(obs_collection=[\"HLA\"], s_dec=[50,60], \n",
+    "                                            calib_level=[3], proposal_pi=\"Mould*\", \n",
+    "                                            dataproduct_type=\"IMAGE\", t_max=[49800,49820])\n",
+    "\n",
+    "#obsByCriteria = Observations.query_criteria(obs_id = 'hst_05766_04_wfpc2_total_wf')"
```

Comment:
I would make this a second cell and say something like "If you know the MAST Observation ID, you can do a direct search using the "obs_id" parameter.  Users should note that other similar-sounding columns ("objID", "obs_id") should NOT be used for most queries as they are primarily internal identifiers.